bsg_dmc/bsg_dmc.vh

```diff
@@ -1,9 +1,18 @@
-
 `ifndef BSG_DMC_VH
 `define BSG_DMC_VH
 
   `include "bsg_defines.v"
 
+  `define declare_dmc_trace_entry_s(addr_width_mp, burst_width_mp, data_width_mp) \
+  	typedef struct packed { \
+        bit [3:0] wdata_cmd_n; \
+        bit [3:0] zeros; \
```

Comment:
Is this padding? If so, put on MSB. Also, is 4 always correct? Probably it should be derived from addr/data width